### PR TITLE
Auto select country calling code by device country code #12

### PR DESCRIPTION
**Pull request by shana0440**
Connect #12 

I using `NativeModules` to get the country code.
It can make this project not rely [react-native-sim-data](https://github.com/pocesar/react-native-sim-data) and keep in this project are pure JS.

### CODE REVIEW

File: packages/react-native-oursky/index.d.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export interface CountryPickerProps extends ExtraTextProps {
   selectedValue?: string;
   backButtonText?: React.ReactNode;
   headerTitle?: React.ReactNode;
+  defaultByDeviceCountry?: boolean;
```
Comment (rickmak) [inline packages/react-native-oursky/index.d.ts]: https://github.com/oursky/react-native-oursky/pull/13#discussion_r239316721

Please make it non-optional.

**Comment by shana0440** [comment]
@cychiuae @rickmak updated thanks.

**Comment by louischan-oursky** [comment]
@cychiuae Do you have any concern on using `rn-sim-info`. Just skimmed through [the source code of it](https://github.com/jojonarte/rn-sim-info) and it seems fine.

**Comment by cychiuae** [comment]
@shana0440 Conflicted

**Comment by shana0440** [comment]
@cychiuae rebased thanks.

**Pull request merged**